src/pages/RepoData/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { BackButton, ProfileInfo } from '../../components';
+
+const RepoData = () => {
+	const { results } = useSelector((state) => state.reposReducer);
+	const { repo } = useParams();
+
+	// Filters all the results using the repo's name and then grabs the only repo object that's in the array
+	const selectedRepo = results.filter((repository) => repository.name === repo)[0];
+	const { name, html_url, stargazers_count, watchers_count, forks_count, language } = selectedRepo;
+
+	return (
+		<>
+			<BackButton />
+			<ProfileInfo />
+			<h1>
+				<a href={html_url}>{name}</a>
+			</h1>
+			<div className="repo-info">
+				<div className="flex-column">
+					<p>Forks: {forks_count}</p>
+				</div>
+				<div className="flex-column">
+					<p>Stargazers: {stargazers_count}</p>
+				</div>
+				<div className="flex-column">
+					<p>Watchers: {watchers_count}</p>
+				</div>
+				<div className="flex-column">
+					<p>Written in {language}</p>
```

Comment:
something annoying about the language from the api is that it only shows the top used language, so maybe change this to "Most Common Language used:" or something. 

If you type in my GitHub username as well, and go onto API-Design. That project is just a readme file so it doesn't use any specific language and therefore does not show anything. This was actually one of the reasons I didnt use the language column